packages/shared/styles/components/atoms/SfColor.scss

```
@@ -2,24 +2,48 @@
 
 .sf-color {
   box-sizing: border-box;
+  position: relative;
   width: var(--color-width, var(--color-size, 2.5rem));
   height: var(--color-height, var(--color-size, 2.5rem));
-  padding: var(--color-padding, 3px);
   background: var(--color-background);
   border: 0;
   border-radius: var(--color-border-radius);
-  &__fill {
-    box-sizing: border-box;
-    width: 100%;
-    height: 100%;
-    border: var(--color-fill-border, 2px solid transparent);
-    border-radius: var(--color-border-radius);
-    transition: border-color 150ms linear;
+  box-shadow: var(--color-box-shadow);
+  transition: transform 150ms linear, box-shadow 150ms linear;
+  cursor: pointer;
+
+  &__badge {
+    --badge-padding: var(--spacer-extra-small);
+    --badge-border-radius: 100%;
+    position: absolute;
+    top: -25%;
+    right: -25%;
+
+    &-enter-active {
+      animation: bounce-in 300ms;
+    }
+
+    &-leave-active {
+      animation: bounce-in 300ms reverse;
+    }
   }
-  &--active,
-  &:hover {
-    --color-fill-border: 2px solid var(--c-white);
+
+  @include for-desktop {
+    --color-width: 0.75rem;
+    --color-height: 0.75rem;
+
+    &:hover,
+    &--active {
+      transform: scale(1.667);
+    }
```

Maybe we can go with: 
```suggestion
    &--active {
      &:hover {
          transform: scale(1.667);
        }
    }
```

---

I think after that it won't work as in designs